### PR TITLE
Added authorization

### PR DESCRIPTION
**Pull request by danylokolohryvko**
History and details moved to separate pages

### CODE REVIEW

File: DepositUI/Pages/Details.razor.csff [hunk 1]
```diff
@@ -0,0 +1,9 @@
+﻿namespace DepositUI.Pages
+{
+    public partial class Details
```
Comment (MaxSqui) [inline DepositUI/Pages/Details.razor.csff]: Why do we need this file?

File: DepositUI/Pages/Calculate.razor.cs [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    {
+        private List<DepositCalc> depositDetails;
+        private DepositModel deposit = new DepositModel();
+
+        [Inject]
+        private IDepositService DepositService { get; set; }
+        [Inject]
+        private IMapper Mapper { get; set; }
+
+        private async Task GetDepositCalc()
```
Comment (MaxSqui) [inline DepositUI/Pages/Calculate.razor.cs]: Use the whole word instead of an abbreviation.

Comment (MaxSqui) [reply DepositUI/Pages/Calculate.razor.cs]: What's the point of calling this method with 'Get' prefix, but returning value is just async operation.

File: DepositUI.BLL/Services/DepositService.cs [hunk 3]
```diff
@@ -59,9 +64,24 @@ public async Task<List<DepositCalcDTO>> CalculateDepositAsync(DepositDTO deposit
             return null;
         }
 
+        public async Task<string> GetDepositCSV(int depositId)
+        {
+            var url = $"{getDepositCSVUrl}?depositId={depositId}";
+            var response = await this.SendRequestAsync(HttpMethod.Get, url);
+            if(response.IsSuccessStatusCode)
```
Comment (MaxSqui) [inline DepositUI.BLL/Services/DepositService.cs:70]: always add an empty line before "if" statement and after

File: DepositUI/Components/NavTabs.razor [hunk 4]
```diff
@@ -0,0 +1,13 @@
+﻿<ul class="nav nav-tabs">
+    <li class="nav-item">
+        <a href="@(Mode != ModeType.Main ? "/deposit" : "")" class="nav-link@(Mode == ModeType.Main ? " active" : "" )">Calculations</a>
+    </li>
+    <li class="nav-item">
+        <a href="@(Mode != ModeType.History ? "/deposit/history" : "")" class="nav-link@(Mode != ModeType.Main ? " active" : "" )">History</a>
+    </li>
+</ul>
+
+@code {
```
Comment (IhorNahorniak) [inline DepositUI/Components/NavTabs.razor]: Keep at split files convention

File: DepositUI.BLL/Services/DepositService.cs [hunk 5]
```diff
@@ -1,28 +1,33 @@
...
     public class DepositService : IDepositService
     {
-        private readonly string depositsUrl = "https://localhost:44320/api/deposit";
-        private readonly string depositCalculationUrl = "https://localhost:44320/api/depositcalculation";
-        private readonly string calculateDepositUrl = "https://localhost:44320/api/calculate";
+        private readonly string depositsUrl;
+        private readonly string depositCalculationUrl;
+        private readonly string calculateDepositUrl;
+        private readonly string getDepositCSVUrl;
         private readonly HttpClient client;
```
Comment (IhorNahorniak) [inline DepositUI.BLL/Services/DepositService.cs:16]: On the future, please, use full naming for everything, don't skimp on letters :)

**Pull request merged**